Library/Homebrew/extend/ENV.rbi

```diff
@@ -10,6 +10,10 @@ class Sorbet
     module Static
       class ENVClass
         include EnvActivation
+        # Note: This is a bit misleading, as at most only one of these can be true
+        # See: EnvActivation#activate_extensions!
+        include Stdenv
+        include Superenv
```

Comment:
These are technically unsound, as the comment indicates, but I argue that they are a reasonable tradeoff for readability.